assets/Gruntfile.js

```diff
@@ -81,13 +81,45 @@ module.exports = function (grunt) {
           open: true,
           middleware: function (connect) {
             return [
-              modRewrite(['!^/(config.js|favicon.ico|(bower_components|scripts|images|styles|views)(/.*)?)$ /index.html [L]']),
+              modRewrite(['!^/(config.js|favicon.ico|(bower_components|scripts|images|styles|views|console|console/java)(/.*)?)$ /index.html [L]']),
+              connect().use('/config.js', function(req, res, next) {
+                var KUBERNETES_HOST = process.env.KUBERNETES_HOST || 'localhost';
+                var KUBERNETES_PORT = process.env.KUBERNETES_PORT || '8443';
+                var KUBERNETES_PROXY_VERSION = process.env.KUBERNETES_PROXY_VERSION || 'v1beta3';
+
+                var answer = 'window.OPENSHIFT_CONFIG = {' +
```

Comment:
Kinda stinks though when you just want to run the console in dev mode against an instance running elsewhere.  I can take it out I guess.  Though ideally you'd maybe just read config.js off the disk and allow environment variables to overwrite defaults in config.js.  Anyway, will take it out.